jl4-core/src/L4/Parser.hs

```diff
@@ -1131,6 +1132,23 @@ ifthenelse = do
       <*  annoLexeme (spacedToken_ TKElse)
       <*> annoHole (indentedExpr current)
 
+multiWayIf :: Parser (Expr Name)
+multiWayIf = do
+  current <- Lexer.indentLevel
+  attachAnno $
+    MultiWayIf emptyAnno
+      <$  annoLexeme (spacedToken_ TKBranch)
+      <*> many do
+        (,)
+          <$> do
+             annoLexeme (spacedToken_ TKIf)
+              *> annoHole (indentedExpr current)
+          <*> do
+             annoLexeme (spacedToken_ TKThen)
+              *> annoHole (indentedExpr current)
+      <*  annoLexeme (spacedToken_ TKOtherwise)
+      <*> annoHole (indentedExpr current)
+
```

Comment:
The use of `many` feels weird, it should likely still be `indented` in some way, otherwise the following is accepted:

![image](https://github.com/user-attachments/assets/b8163def-aa29-4fe8-a3de-d83c254c2288)

Which looks like a bug to me